CMakeLists.txt

```diff
@@ -114,6 +114,11 @@ if(ITKPythonPackage_SUPERBUILD)
         )
     endif()
 
+    if(${CMAKE_VERSION} VERSION_LESS 3.24)
+      set(download_extract_timestamp_flag)
+    else()
+      set(download_extract_timestamp_flag DOWNLOAD_EXTRACT_TIMESTAMP TRUE)
+    endif()
```

Comment:
```suggestion
```

Reply:
Add instead the following around line 44 (before `set(ep_common_cmake_cache_args )`)


```
set(ep_download_extract_timestamp_arg )
if(CMAKE_VERSION VERSION_EQUAL "3.24" OR CMAKE_VERSION VERSION_GREATER "3.24")
  # See https://cmake.org/cmake/help/latest/policy/CMP0135.html
  set(ep_download_extract_timestamp_arg DOWNLOAD_EXTRACT_TIMESTAMP 1)
endif()
```

then consistently update all `ExternalProject_Add` to make use of it.

Reply:
My understanding is that this setting should only be applied to archives, i.e. not Git repositories, correct?

Reply:
Then, when the minimum required version of CMake will be 3.24, the corresponding will be just removed.

Reply:
ok, added